fronts-client/src/actions/OptionsModal.ts

```diff
@@ -15,7 +15,8 @@ const startOptionsModal = (
     description,
     options,
     onCancel,
-    showCancelButton
+    showCancelButton,
+    isOpen: true
```

Comment:
Had to add this for the type to conform. Guessed it must be true if the modal is being shown?